tests/Unit/Domain.LinnApps.Tests/PurchaseOrderServiceTests/WhenSendingPdfEmail.cs

```diff
@@ -0,0 +1,92 @@
+﻿namespace Linn.Purchasing.Domain.LinnApps.Tests.PurchaseOrderServiceTests
+{
+    using System.Collections.Generic;
+    using System.IO;
+
+    using FluentAssertions;
+
+    using Linn.Purchasing.Domain.LinnApps.PurchaseOrders;
+    using Linn.Purchasing.Domain.LinnApps.PurchaseOrders.MiniOrders;
+
+    using NSubstitute;
+
+    using NUnit.Framework;
+
+    public class WhenSendingPdfEmail : ContextBase
+    {
+        private readonly int employeeNumber = 33107;
+
+        private readonly int orderNumber = 5678;
+
+        private readonly string supplierEmail = "seller@wesellthings.com";
+
+        private MiniOrder miniOrder;
+
+        private ProcessResult result;
+
+        [SetUp]
+        public void SetUp()
+        {
+            this.miniOrder = new MiniOrder { OrderNumber = this.orderNumber };
+
+            this.EmployeeRepository.FindById(this.employeeNumber).Returns(
+                new Employee
+                    {
+                        Id = this.employeeNumber,
+                        FullName = "mario",
+                        PhoneListEntry = new PhoneListEntry { EmailAddress = "mario@karting.com" }
+                    });
+
+            this.EmailService.SendEmail(
```

Comment:
is this supposed to be mocking out a return with .Returns() ? If not its not doing anything and can possibly go?